core/domain/takeout_service_test.py

```diff
@@ -1029,3 +1235,26 @@ def test_export_data_nontrivial(self):
                 expected_images[i].image_export_path,
                 observed_images[i].image_export_path
             )
+
+        profile_takeout_object = takeout_service.export_data_for_user(
+            self.PROFILE_ID_1)
+        observed_data = profile_takeout_object.user_data
+        observed_images = profile_takeout_object.user_images
+        self.assertItemsEqual(observed_data, expected_profile_data)
+        observed_json = json.dumps(observed_data)
+        expected_json = json.dumps(expected_profile_data)
+        self.assertItemsEqual(
+            json.loads(observed_json), json.loads(expected_json))
+
+    def test_export_all_profiles_data_for_user_with_profiles_is_correct(self):
+        pass
+
+    def test_export_all_profiles_data_user_with_no_profiles_is_correct(self):
+        pass
```

Comment:
Are these tests still pending?

Reply:
I mentioned it in [this](https://github.com/oppia/oppia/pull/10241#issuecomment-673113478) comment --can you please say something about that?

Reply:
See my comment above, we should add a test of exporting a profile users data (will be similar to the full user test but with limited data compared to a full user)

Reply:
Done. Added the test here for profiles.